### PR TITLE
step-1-display-stop-ID

### PR DESCRIPTION
**Pull request by rulles13**

### CODE REVIEW

File: app/loaders/gtfsDepartures.tsx [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export default async function searchTenRoutes(params: any) {
```
Comment (heleneruelle) [inline app/loaders/gtfsDepartures.tsx]: careful with the name

File: app/pages/SmartDisplay.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+function SmartDisplay() {
+
+    const data = useLoaderData()
+  
+    console.log(data);
+  
+    return (
+      <div>
+        <h1>Le stop ID est : </h1>
+        <h2>{data.params.stopIds}</h2>
```
Comment (heleneruelle) [inline app/pages/SmartDisplay.tsx]: Could you deconstruct the data obj to access `stopIds` ? 

Also using a ternary operator (https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Operators/Conditional_Operator#:~:text=L'op%C3%A9rateur%20(ternaire)%20conditionnel,%2Fif...else%20.) can you try to display : 
if (stopIds) ==> (
```
<h1>Le stop ID est : </h1>
<h2>{data.params.stopIds}</h2>
```
)
if (!stopIds) ==> (
`<h1>Error Message</h1>`
)

**Review by heleneruelle — CHANGES_REQUESTED** [review]

File: app/loaders/gtfsDepartures.tsx (hunk 1 above)
Comment (rulles13) [reply app/loaders/gtfsDepartures.tsx]: I will change the name into :"displayStopIds"

File: app/pages/SmartDisplay.tsx (hunk 2 above)
Comment (rulles13) [reply app/pages/SmartDisplay.tsx]: I can try

**Review by heleneruelle — APPROVED** [review]

**Pull request merged**